src/webgpu/shader/execution/expression/call/builtin/subgroupElect.spec.ts

```diff
@@ -0,0 +1,384 @@
+export const description = `
+Execution tests for subgroupElect
+`;
+
+import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../../../common/util/data_tables.js';
+import { iterRange } from '../../../../../../common/util/util.js';
+
+import {
+  SubgroupTest,
+  kFramebufferSizes,
+  getUintsPerFramebuffer,
+  kWGSizes,
+  kDataSentinel,
+  runComputeTest,
+  runFragmentTest,
+  kPredicateCases,
+} from './subgroup_util.js';
+
+export const g = makeTestGroup(SubgroupTest);
+
+/**
+ * Checks subgroupElect compute shader results
+ *
+ * @param metadata An array of uint32s containing:
+ *                 * subgroup_invocation_id in first half
+ *                 * subgroup_size in second half
+ * @param output An array of uint32s containing elect results
+ * @param filter A functor to determine active invocations
+ */
+function checkCompute(
+  metadata: Uint32Array,
+  output: Uint32Array,
+  filter: (id: number, size: number) => boolean
+): Error | undefined {
+  const size = metadata[output.length];
+  let elected = 129;
+  for (let i = 0; i < 128; i++) {
+    if (filter(i, size)) {
+      elected = i;
+      break;
```

Comment:
for my own notes:  this break, and the increasing index, means we pick the lowest numbered invocation.
That's per spec.